tensorflow/core/framework/common_shape_fns.cc

```diff
@@ -707,7 +706,18 @@ Status Conv2DShapeImpl(shape_inference::InferenceContext* c,
     TF_RETURN_IF_ERROR(CheckValidPadding(padding, explicit_paddings,
                                          /*num_dims=*/4, data_format));
   } else {
+    std::vector<int64_t> p_list;
+    Status s_p_list = c->GetAttr("padding_list", &p_list);
     CHECK(padding != Padding::EXPLICIT);  // Crash ok.
+    if (!s_p_list.ok() && !errors::IsNotFound(s_p_list)) {
+      return s_p_list;
+    }
+    if (s_p_list.ok() && !p_list.empty()) {
+      padding = Padding::EXPLICIT;
+      explicit_paddings = p_list;
```

Comment:
Please add comments explaining that fused int8 conv ops use this field to support explicit padding.